### ISSUE TITLE
gherkin-c ninja build fails

### ISSUE THREAD

**Issue by DewJunkie** [cucumber/cucumber#1200]
## Summary

ninja is the default build system of MSVC devenv
If build is attempted with ninja, build fails
```
gherkin\c\out\build\x64-Debug (default)\ninja : error : build.ninja:616: bad $-escape (literal $ must be written as $$)
```

## Expected Behavior

Build should succeed, with no changes needed to any project files

## Current Behavior

CmakeLists.txt::134 is the source of this failure
```
add_custom_target(install_${PROJECT_NAME}
        $(MAKE) install
        DEPENDS gherkin
        COMMENT "Installing ${PROJECT_NAME}")
```
I don't know what $(MAKE) install is supposed to be doing, but build can't continue with that line in place.  I tried sugested solution of escapting with $$, tried escaping with \$.

## Possible Solution
Omit this step if build system is ninja.  I will look into what $(MAKE) results in on linux, and maybe can propose better solution.
```
if(NOT ${CMAKE_GENERATOR} STREQUAL "Ninja")
    add_custom_target(install_${PROJECT_NAME}
            $(MAKE) install
            DEPENDS gherkin
            COMMENT "Installing ${PROJECT_NAME}")
endif()
```

## Steps to Reproduce (for bugs)
open folder gherkin/c in msvc
attempt build

## Context & Motivation

Build on all the things

## Your Environment

* Version used: msvc 2019 16.8
* Operating System and version: Win10
* Link to your project:

**Comment by aslakhellesoy** [comment]
We don't currently have a regular maintainer of `gherkin/c` - pull requests welcome!

**Comment by DewJunkie** [comment]
Fixed by PR #1201
